README.md

```diff
@@ -64,8 +64,12 @@ module.exports = {
 
 2. options?: object
 
-   - level: number => Level of the current section (0 = main section)
-   - headline: string => Headline for the current section
+   - level?: number => Level of the current section (0 = main section)
+   - headline?: string => Headline for the current section
+   - showHelp?: string => If you want to show the `help` command at the top
+   - backLabel?: string => custom label to indicate `go back` action
+   - cancelLabel?: string => custom label to indicate `cancel` action
+   - byeLabel?: string => custom `bye` label
```

Comment:
For bye, message fits better than label, since it is not the label of a menu item but an output message.

Reply:
Fixed now